SceneKitReel/SceneKitReel.sln

```diff
@@ -11,47 +11,24 @@ Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
 		Release|Any CPU = Release|Any CPU
-		AppStore|Any CPU = AppStore|Any CPU
 		Debug|iPhoneSimulator = Debug|iPhoneSimulator
 		Release|iPhoneSimulator = Release|iPhoneSimulator
 		Debug|iPhone = Debug|iPhone
 		Release|iPhone = Release|iPhone
-		Ad-Hoc|iPhone = Ad-Hoc|iPhone
-		AppStore|iPhone = AppStore|iPhone
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{CFDC4C7E-92F4-4BB6-A311-0317BA4586AB}.Ad-Hoc|iPhone.ActiveCfg = Debug|Any CPU

```

Comment:
Build Configuration for Mac sample needs to be fixed. iOS and shared samples are defaulting to the correct configurations when user selects respective projects.  Mac project should be Debug|MyMac and Release|MyMac.
https://screencast.com/t/rDLngplRLa